examples/mssqlclient.py

```diff
@@ -105,10 +106,14 @@
         res = False
     if res is True:
         shell = SQLSHELL(ms_sql, options.show)
-        if options.file is None:
-            shell.cmdloop()
-        else:
+        if options.file:
             for line in options.file.readlines():
                 print("SQL> %s" % line, end=' ')
                 shell.onecmd(line)
+        elif options.cmd:
+            for c in options.cmd:
+                print("SQL> %s" % c)
+                shell.onecmd(c)
```

Comment:
Changed parameter name

```suggestion
        elif options.command:
            for c in options.command:
                print("SQL> %s" % c)
                shell.onecmd(c)
```